Editor/Data/Type/DefaultObservableListDrawer.cs

```diff
@@ -45,7 +47,14 @@ public override void OnGUI(Rect position, SerializedProperty property, GUIConten
                 using (new EditorGUI.IndentLevelScope())
                 {
                     dynamic list = propertyObject;
-                    bool listIsEmpty = list.NonSubscribableElements.Count == 0;
+                    dynamic elements = list.NonSubscribableElements;
+                    bool listIsEmpty = elements.Count == 0;
```

Comment:
```suggestion
                    bool isListEmpty = elements.Count == 0;
```
I overlooked this one in the previous PR.